extra-utils/premake/autobuild/build

```
@@ -0,0 +1,9 @@
+# build script copied from Arch Linux
+abinfo "Building premake ..."
+cd "$SRCDIR/build/gmake2.unix"
+make STRIP="true" config=release
+
+abinfo "Installing premake and LISENSE file"
+cd "$SRCDIR"
+install -v -Dm755 "bin/release/premake5" "${PKGDIR}/usr/bin/premake5"
```

Use absolute path everywhere.
```suggestion
install -v -Dm755 "$SRCDIR/bin/release/premake5" "${PKGDIR}/usr/bin/premake5"
```